canonical-web-and-design/hiring.canonical.com
# Add charts to dashboard

## Done
- "Age of pending applications" and "Performance overview" charts are implemented.
- User's full name is retrieved from an endpoint and displayed.
- Number of pending applications are displayed.

## QA
Setup:
- Run server as Peter: `dotrun --env DEBUG_USER_EMAIL=peter.mahnke@canonical.com`

Validation:
- View charts at http://0.0.0.0:8501/
- Verify full name is displayed. (it may not be correct if you're not login with Peter's credentials. It should display the person's name who logged in.)
- Verify number of applications that are pending.
- Verify the "Age of pending applications" chart is displayed.
- Verify the "Performance overview" chart is displayed

## Issue
Fixes #121 #119 #124
